Core/GDCore/IDE/Project/ProjectResourcesCopier.h

```diff
@@ -39,14 +39,16 @@ class GD_CORE_API ProjectResourcesCopier {
    * of the resources will be preserved when copying. Otherwise, everything will
    * be send in the destinationDirectory.
    *
-   * \return true if no error happened
+   * \return The list of the exported names of all resources, useful for
+   * prefetching and caching.
    */
-  static bool CopyAllResourcesTo(gd::Project& project,

```

Comment:
I checked and calls to this function was using the boolean return value (which always returned true either way)